src/Runtime/SymbolResolver.cs

```diff
@@ -55,11 +55,36 @@ static SymbolResolver ()
             }
         }
 
-        public static IntPtr LoadImage (ref string name)
+        public static IntPtr LoadImage (ref string name, string searchdir = null)
         {
+            if (searchdir == null)
```

Comment:
As with my remark about ResolveSymbol, please convert this to a local variable in case we don't need the parameter.